packages/core/README.md

```diff
@@ -43,7 +43,7 @@ Choose _one_ theme from the following options:
 `@use` Castor within your main Sass file:
 
 ```scss
-@use '~@onfido/castor';
+@use '@onfido/castor';
```

Comment:
Do you think such change on docs is needed?

So most likely users will have a setup that is not requiring a `~`?

Reply:
I was thinking we should remove `~` from everywhere except our examples where actually need them.

It's a requirement of `webpack` and `Parcel 1` but all other bundlers, including `Parcel 2`, can't actually parse the tilde without an alias.

Reply:
Nevermind, it seems it's still needed, so let's keep it for now.